pkg/utils/emails/contacts.go

```diff
@@ -0,0 +1,72 @@
+package emails
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/sendgrid/sendgrid-go"
+)
+
+const (
+	sgHost     = "https://api.sendgrid.com"
+	sgContacts = "/v3/marketing/contacts"
+)
+
+type AddContact struct {
+	ListIDs  []string   `json:"list_ids"`
+	Contacts []*Contact `json:"contacts"`
+}
+
+type Contact struct {
+	FirstName    string        `json:"first_name"`
+	LastName     string        `json:"last_name"`
+	Email        string        `json:"email"`
+	Country      string        `json:"country"`
+	CustomFields *CustomFields `json:"custom_fields"`
+}
+
+// TODO: make custom fields request to get field IDs rather than hardcoding.
+type CustomFields struct {
+	Title                string `json:"e1_T"`
+	Organization         string `json:"e2_T"`
+	CloudServiceProvider string `json:"e3_T"`
+}
+
+// AddContact adds a contact to the SendGrid marketing contacts list.
+func (m *EmailManager) AddContact(contact *Contact) error {
```

Comment:
In the future we may want to be able to specify which list IDs the contact is added to.

```suggestion
// AddContact adds a contact to the SendGrid marketing contacts list.
// TODO: allow user to specify marketing list IDs using variadic arguments
func (m *EmailManager) AddContact(contact *Contact) error {
```